src/main/java/net/greatstart/services/UserService.java

```diff
@@ -41,4 +41,11 @@ public User getUserById(long id) {
         return userDao.getAll();
     }
 
+    // TODO : implement this

```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)